debian/bionic/foreman/rules

```diff
@@ -33,9 +33,14 @@ build:
 	dh $@ --with=systemd
 
 override_dh_installinit:
-	dh_installinit
+	dh_installinit --only-scripts
 
 override_dh_systemd_enable:
-	dh_systemd_enable --package=foreman-service
+	dh_systemd_enable --name=foreman.service --package=foreman-service
+	dh_systemd_start --no-start --name=foreman.service --package=foreman-service
```

Comment:
these need to go into a `override_dh_systemd_start` target, otherwise you end up calling 'em twice.